lib/trsort.c

```diff
@@ -328,13 +328,13 @@ tr_introsort(saidx_t *ISA, const saidx_t *ISAd,
              saidx_t *SA, saidx_t *first, saidx_t *last,
              trbudget_t *budget) {
 #define STACK_SIZE TR_STACKSIZE
-  struct { const saidx_t *a; saidx_t *b, *c; saint_t d, e; }stack[STACK_SIZE];
```

Comment:
stack[x].e stored the trlink but the refactoring makes it unneccessary. It seems trlink was used to track when to switch to heap sort in case of tandem repeat.